front/components/sparkle/AppLayout.tsx

```diff
@@ -355,7 +355,7 @@ export default function AppLayout({
 
           <main
             className={classNames(
-              "h-full overflow-x-hidden pt-16",
+              "h-full pt-16",
```

Comment:
I'm not sure I understand this one?

Reply:
I am just removing the thing that breaks autoscroll

Reply:
With these kind of layout question we really want to be methodological. Local minimum finding will inevitably lead to something that is not maintainable.